lib/insights/api/common/rbac/access.rb

```diff
@@ -25,6 +25,23 @@ def accessible?
             @acl.any?
           end
 
+          def is_accessible?(app_name, resource, verb)
```

Comment:
@bzwei The is_accessible is the new method which will replace the accessible? method. The current accessible? method doesn't take any arguments. The is_accessible? takes the app name, resource, verb.

Reply:
Since the gem is version controlled, we can safely replace the existing `accessible?` method with the new implementation. Any app that uses this gem will update the use of `accessible?` and bump up the gem version number. We don't need to have another round to rename `is_accessible?` to `accessible?`.

Reply:
Sorry, depend on how the gem version is specified in Gemfile, we do need to maintain backward compatibility.
You may want to consider to have `def accessible?(app_name = nil, resource = nil, verb = nil)` for a while for backward compatibility purpose. And in the future remove the default values for arguments. This way at least both catalog-api and approval-api does not need to switch from `is_accessible?` to `accessible?`.